client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from 'react';
+import { Route, Switch, Link, useHistory, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { LoadingSpinner, Section, SectionTableWrapper } from '_common';
+import * as ROUTES from '../../constants/routes';
+import { Sidebar, SectionMessage } from '_common';
+import Sysmon from '../SystemMonitor';
+import { SystemStatusQueueTable } from './SystemStatusQueueTable/SystemStatusQueueTable';
+
+import './SystemStatus.global.css';
+import styles from './SystemStatus.module.scss';
+
+const root = `${ROUTES.WORKBENCH}${ROUTES.SYSTEM_STATUS}`;
+
+const SystemStatusSidebar = ({ systemList }) => {
+  var sidebarItems = [];
+
+  systemList.forEach((system) => {
+    sidebarItems.push({
+      to: `${root}/${system.hostname}`,
+      label: `${system.display_name}`,
+      disabled: false,
+      hidden: false,
+      iconName: '',
+    });
+  });
+
+  return <Sidebar sidebarItems={sidebarItems} />;
+};
+
+const Layout = ({ hostname }) => {
+  const {
+    loading,
+    error: loadingError,
+    list: systemList,
+  } = useSelector((state) => state.systemMonitor);
+
+  const selectedSystem = systemList.find((sys) => sys.hostname === hostname);
+
+  if (loadingError) {
+    return (
+      <Section className={styles['no-results-message']}>
+        <SectionMessage type="info">
+          Unable to gather system information
+        </SectionMessage>
+      </Section>
+    );
+  }
+
+  return (
+    <>
+      <Section
+        className={styles['section']}
+        contentLayoutName="oneColumn"
```

Comment:
**Potential Alternative**
Please try other layouts e.g. `contentLayoutName="twoColumn"`?

You gave extra space on the right of the right of this section's content via [`width: 50%`](https://github.com/TACC/Core-Portal/pull/807/files#diff-e1a6008073beb0bfd6a59ba813f88ddc87022f33cbee96241f0890f845efd193R24).

Such a layout looks like `twoColumns`. 🙂

Reply:
I am trying this. It does not work immediately. I investigate.

Reply:
I succeeded in #816.